l10n_it_website_sale_corrispettivi/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Italian localization - Website Sale Corrispettivi',
```

Comment:
localization → Localization
Visto che non è traducibile, il nome meglio metterlo in italiano (e Website → sito web)